corehq/apps/locations/static/locations/ko/location_drilldown.async.js

```diff
@@ -143,7 +143,11 @@ function LocationModel(data, root, depth, func, withAllOption) {
 
   // helpers to account for the 'all' meta-entry
   this.num_children = ko.computed(function() {
-      return (this.children().length === 0 ? 0 : this.children().length - 1);
+      var length = this.children().length;
+      if (withAllOption) {
+          length -= 1;
```

Comment:
It seems to me that with 'all' entry it should be always greater than 0 but I don't know original purpose of this so I will revert my last commit.